specification/billing/resource-manager/Microsoft.Billing/stable/2021-10-01/billingSubscription.json

```diff
@@ -16,30 +16,27 @@
     "application/json"
   ],
   "paths": {
-    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/customers/{customerName}/billingSubscriptions": {
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/billingSubscriptions": {
       "get": {
         "tags": [
           "BillingSubscriptions"
         ],
-        "operationId": "BillingSubscriptions_ListByCustomer",
-        "description": "Lists the subscriptions for a customer. The operation is supported only for billing accounts with agreement type Microsoft Partner Agreement.",
+        "operationId": "BillingSubscriptions_ListByBillingAccount",
+        "description": "Lists the subscriptions for a billing account. The operation is supported for billing accounts with agreement type Microsoft Customer Agreement, Microsoft Partner Agreement or Enterprise Agreement.",
```

Comment:
Lists the subscriptions for a billing account. The operation is supported for billing accounts with agreement type Microsoft Customer Agreement, Microsoft Partner Agreement and Enterprise Agreement.",